assets/base/i18n/en.json

```diff
@@ -1,1686 +1,37 @@
 {
-  "about.buildnumber": "Build Number:",
```

Comment:
Is this going to be fighting with the translation server PRs constantly to remove strings that aren't used in the mobile app? Maybe we should only remove entries from this file as part of the release process so that it won't conflict and so that we also have a good list of existing strings when adding new mobile features

Reply:
Not really, the tooling to merge/split the translations is already done and working, @enahum will need to replace the current tooling with the new tooling, thats it :)